plugins/woocommerce/package.json

```diff
@@ -418,7 +418,6 @@
 				"node_modules/@woocommerce/e2e-core-tests/CHANGELOG.md",
 				"node_modules/@woocommerce/api/dist/",
 				"node_modules/@woocommerce/admin-e2e-tests/build",
-				"node_modules/@woocommerce/classic-assets/build",
```

Comment:
I'm not sure why this line keeps getting removed when I run `pnpm i`. We should restore this line.

Reply:
@rrennick dependencyOutputs is auto generated by the .pnpmfile.cjs if it's removing this there may be a bug there to investigate.